lib/idp_common_pkg/idp_common/assessment/granular_service.py

```diff
@@ -2018,4 +2018,4 @@ def _convert_error_list_to_string(self, errors) -> str:
             return f"{first_errors} and {len(errors) - 1} more errors"
         else:
             # Few errors - show all
-            return "; ".join(errors)
```

Comment:
Seems to be a lot of line removed / identical line added - which makes the PR harder to review..